src/ic-websocket-gateway/src/canister_poller.rs

```diff
@@ -214,8 +222,11 @@ impl CanisterPoller {
             *self.message_nonce.read().await,
             first_client_key,
         );
-
-        if canister_result.messages.len() > 0 {
+        let number_of_returned_messages = canister_result.messages.len();
+        if number_of_returned_messages > 0 {
+            if number_of_returned_messages >= MAX_NUMBER_OF_RETURNED_MESSAGES {
```

Comment:
So this way if the developer sets a max number of returned messages > 30 on the CDK we continue printing this warn?